scripts/python/airflow/dags/contralory_get_pdf.py

```diff
@@ -0,0 +1,96 @@
+from datetime import timedelta
+from airflow.models import DAG
+from airflow.models import Variable
+from airflow.utils.dates import days_ago
+from airflow.operators.dummy_operator import DummyOperator
+from airflow.operators.python_operator import PythonOperator
+import os
+
+from contralory.contralory_page import contraloria_get_urls, contraloria_download_pdfs
+from contralory.parse_pdf_name import extract_data_from_names
+from contralory.name_to_database import push_to_postgre
+from contralory import if_all_done
+
+try:
+    target_dir = os.path.join(Variable.get('CGR_PDF_FOLDER'))
```

Comment:
Use two variables, each one with the full path instead of extract two paths from one variable.